src/Sylius/Bundle/CoreBundle/ExpressionLanguage/ChannelContextVariables.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\ExpressionLanguage;
+
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Resource\Symfony\ExpressionLanguage\VariablesInterface;
+
+final readonly class ChannelContextVariables implements VariablesInterface
+{
+    public function __construct(
+        private ChannelContextInterface $channelContext,
+    ) {
+    }
+
+    /**
+     * @return array{sylius_channel_context: ChannelContextInterface}
+     */
+    public function getVariables(): array
+    {
+        return [
+            'sylius_channel_context' => $this->channelContext,
```

Comment:
It could be cool to add sth like that.

```suggestion
            'sylius_channel_context' => $this->channelContext,
            'sylius_channel' => $this->channelContext->getChannel(),
```

That would be less verbose on expressions.

Reply:
or `sylius_current_channel`